### ISSUE TITLE
clang: error: no such file or directory: 'fastBPE/fastBPE.cpp'

### ISSUE THREAD

**Issue by ver007** [glample/fastBPE#60]
clang: error: no such file or directory: 'fastBPE/fastBPE.cpp'
mac os 10.9 read error

**Comment by gordicaleksa** [comment]
+1 on Ubuntu

**Comment by manueldeprada** [comment]
Clone the repo, execute:
g++ -std=c++11 -pthread -O3 fastBPE/main.cc -IfastBPE -o fast
pip install ./

Working on python 3.12, fedora linux.

**Comment by Mad-At-Line** [comment]
Yeah I am also having this error anyone have a solution for windows or WSL?

**Comment by aaron-jencks** [comment]
I have fixed this issue, `setup.py` is looking for an incorrect file https://github.com/glample/fastBPE/pull/66
